Project.toml

```diff
@@ -4,8 +4,12 @@ authors = ["michael catchen <michael.catchen@colorado.edu>"]
 version = "0.1.2"
 
 [deps]
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
```

Comment:
Is DataFrames an actual requirement here, v. simply allowing anything that uses the table interface?